app/services/rbac.rb

```diff
@@ -74,9 +74,14 @@ def structurize(access_entry)
 
     def valid_inventory_groups_definition?(definition)
       # FIXME: JSON.parse needs to be moved to the RBAC API client
-      JSON.parse(definition.value).instance_of?(Array) &&
-        definition.operation == 'in' &&
-        definition.key == 'group.id'
+      begin
+        JSON.parse(definition.value).instance_of?(Array) &&
+          definition.operation == 'in' &&
+          definition.key == 'group.id'
+      rescue JSON::ParserError => e
```

Comment:
I think that the rescue can be moved to the end of the function. `begin` would not be needed then. Although, idk how it will be with the return value.